crates/sui-indexer/src/models/events.rs

```diff
@@ -53,7 +53,7 @@ pub struct StoredEvent {
     #[diesel(sql_type = diesel::sql_types::Text)]
     pub event_type: String,
 
-    #[diesel(sql_type = diesel::sql_types::Bytea)]
+    #[diesel(sql_type = diesel::sql_types::Binary)]
```

Comment:
Why do we need to change this?

Reply:
`Bytea` is defined in `diesel/postgres` but Binary is defined in `diesel`. But `Bytea` is nothing but a type alias for binary `pub type Bytea = Binary` , so using Binary makes it possible to be used across both postgres and mysql code paths